Broken for checkstyle 8.5

In Eclipse:

```
com.puppycrawl.tools.checkstyle.api.CheckstyleException: SuppressionCommentFilter is not allowed as a child in Checker
    at com.puppycrawl.tools.checkstyle.Checker.setupChild(Checker.java:484)
    at com.puppycrawl.tools.checkstyle.api.AutomaticBean.configure(AutomaticBean.java:186)
    at net.sf.eclipsecs.core.builder.CheckerFactory.createCheckerInternal(CheckerFactory.java:299)
    at net.sf.eclipsecs.core.builder.CheckerFactory.createChecker(CheckerFactory.java:133)
    at net.sf.eclipsecs.core.builder.Auditor.runAudit(Auditor.java:141)
    at net.sf.eclipsecs.core.builder.CheckstyleBuilder.handleBuildSelection(CheckstyleBuilder.java:306)
    at net.sf.eclipsecs.core.builder.CheckstyleBuilder.build(CheckstyleBuilder.java:172)
    at org.eclipse.core.internal.events.BuildManager$2.run(BuildManager.java:735)
    at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
    at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:206)
    at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:246)
    at org.eclipse.core.internal.events.BuildManager$1.run(BuildManager.java:301)
    at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
    at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:304)
    at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:360)
    at org.eclipse.core.internal.events.BuildManager.build(BuildManager.java:383)
    at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:144)
    at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:235)
    at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
```

Maven command line:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.507 s
[INFO] Finished at: 2018-01-08T16:30:44+01:00
[INFO] Final Memory: 27M/275M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle-main) on project parent: Failed during checkstyle configuration: SuppressionCommentFilter is not allowed as a child in Checker -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Docs says that it should be within the `TreeWalker`, not the `Checker`:
http://checkstyle.sourceforge.net/config_filters.html

--- a/src/main/resources/viskan-checkstyle-main.xml
+++ b/src/main/resources/viskan-checkstyle-main.xml
@@ -7,11 +7,6 @@
 
     <property name="severity" value="warning" />
 
-    <module name="SuppressionCommentFilter">
-        <property name="offCommentFormat" value="CSOFF" />
-        <property name="onCommentFormat" value="CSON" />
-    </module>
-
     <!-- See http://checkstyle.sourceforge.net/config_javadoc.html#PackageHtml -->
     <!-- Checks that a package.html file exists for each package. -->
     <!-- <property name="fileExtensions" value="java"/> -->
@@ -26,7 +21,9 @@
 
         <property name="tabWidth" value="4" />
 
-        <module name="FileContentsHolder">
+        <module name="SuppressionCommentFilter">
+            <property name="offCommentFormat" value="CSOFF" />
+            <property name="onCommentFormat" value="CSON" />
         </module>
 
         <!-- [aab] add in order to check whitespaces -->
@@ -334,12 +331,10 @@
         <!-- See http://checkstyle.sourceforge.net/config_blocks.html -->
         <!-- Checks for the placement of left curly braces ('{') for code blocks. -->
         <!-- <property name="option" value="eol"/> -->
-        <!-- <property name="maxLineLength" value="80"/> -->
         <!-- <property name="tokens" value="CLASS_DEF, CTOR_DEF, INTERFACE_DEF, LITERAL_CATCH, LITERAL_DO, LITERAL_ELSE, LITERAL_FINALLY, LITERAL_FOR, LITERAL_IF, LITERAL_SWITCH, LITERAL_SYNCHRONIZED, 
             LITERAL_TRY, LITERAL_WHILE, METHOD_DEF"/> -->
         <module name="LeftCurly">
             <property name="option" value="nl" />
-            <property name="maxLineLength" value="120" />
         </module>
 
         <!-- See http://checkstyle.sourceforge.net/config_blocks.html -->

--- a/src/main/resources/viskan-checkstyle-test.xml
+++ b/src/main/resources/viskan-checkstyle-test.xml
@@ -7,11 +7,6 @@
 
     <property name="severity" value="warning" />
 
-    <module name="SuppressionCommentFilter">
-        <property name="offCommentFormat" value="CSOFF" />
-        <property name="onCommentFormat" value="CSON" />
-    </module>
-
     <!-- See http://checkstyle.sourceforge.net/config_javadoc.html#PackageHtml -->
     <!-- Checks that a package.html file exists for each package. -->
     <!-- <property name="fileExtensions" value="java"/> -->
@@ -26,7 +21,9 @@
 
         <property name="tabWidth" value="4" />
 
-        <module name="FileContentsHolder">
+        <module name="SuppressionCommentFilter">
+            <property name="offCommentFormat" value="CSOFF" />
+            <property name="onCommentFormat" value="CSON" />
         </module>
 
         <!-- [aab] add in order to check whitespaces -->
@@ -310,10 +307,9 @@
         <!-- See http://checkstyle.sourceforge.net/config_blocks.html -->
         <!-- Checks for the placement of left curly braces ('{') for code blocks. -->
         <!-- <property name="option" value="eol"/> -->
-        <!-- <property name="maxLineLength" value="80"/> -->
         <!-- <property name="tokens" value="CLASS_DEF, CTOR_DEF, INTERFACE_DEF, LITERAL_CATCH, LITERAL_DO, LITERAL_ELSE, LITERAL_FINALLY, LITERAL_FOR, LITERAL_IF, LITERAL_SWITCH, LITERAL_SYNCHRONIZED, 
             LITERAL_TRY, LITERAL_WHILE, METHOD_DEF"/> -->
-        <!-- <module name="LeftCurly"> <property name="option" value="nl"/> <property name="maxLineLength" value="120"/> </module> -->
+        <!-- <module name="LeftCurly"> <property name="option" value="nl"/> </module> -->
 
         <!-- See http://checkstyle.sourceforge.net/config_blocks.html -->
         <!-- Checks for braces around code blocks. -->

